manifests/alias.pp

```diff
@@ -7,45 +7,20 @@
 #     }
 #
 define kmod::alias(
-  $source,
-  $ensure = 'present',
-  $file   = '/etc/modprobe.d/aliases.conf',
+  $modulename,
```

Comment:
Yes, it's a breaking change - but it makes more sense - you can't alias something to nothing so it should be mandatory